scripts/development/m3_stack/docker-compose.yml

```diff
@@ -107,5 +107,23 @@ services:
     networks:
       - backend
     image: m3grafana:latest
+  jaeger:
+    networks:
+      - backend
+    build:
```

Comment:
more a question: can you skip the `build:` section here?

Reply:
It appears so..